requirements.txt

```diff
@@ -1,4 +1,4 @@
-qiskit-terra>=0.23.1
+qiskit-terra[qasm3-import]>=0.23.1
 requests>=2.19
```

Comment:
I think this will have to be added to `setup.py` as well

Reply:
Thanks! I added it in 246985e69bc72bc0210d226cac57cc411892910f